### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by lietu**
We need to come up with a way to do these merges back and forth without constantly changing `README.md` and `settings.py`

### CODE REVIEW

File: .github/workflows/build-and-upload.yaml [hunk 1]
```diff
@@ -29,29 +29,30 @@ jobs:
...
-          # Update the DB so we can skip it later
-          trivy image --download-db-only
-
-      - name: Trivy security scan
-        run: |
-          IMAGES=$(poetry run list)
-          for image in $IMAGES; do
-            trivy image --skip-update --severity HIGH,CRITICAL --exit-code 1 "$image"
-          done
+      #      - name: Install Trivy
```
Comment (fbjorn) [inline .github/workflows/build-and-upload.yaml:32]: why not just remove it? you can restore it from commit history when needed

File: .src/docker_build/main.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 )
 from docker_build.validation import validate
 
+from settings import conf
```
Comment (fbjorn) [inline .src/docker_build/main.py:11]: seems like `isort` hook is not working properly, cause `settings` should be located within the same block wit `docker_build`, and `typer` is a 3rd party. it's configurable via `setup.cfg`, but you could skip it for now, just FYI

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 Builds your Docker images automatically, like magic. Good for handling common base images for all your projects, apps,
 whatever.
 
-This repository builds images to [hub.docker.com/u/digitallivinginternational](https://hub.docker.com/u/digitallivinginternational)
+This repository builds images to [hub.docker.com/u/lietu](https://hub.docker.com/u/lietu)
```
Comment (fbjorn) [inline README.md:6]: it should be `digitalliving`, right?

File: README.md [hunk 4]
```diff
@@ -47,3 +48,11 @@ re-thinking or if you should just make a fork for yourself.
 ## If I use this it means you own my things, right?
 
 No. You are responsible for and own your own things. This code is licensed under the [BSD 3-clause license](LICENSE.md).
+
+# Financial support
```
Comment (fbjorn) [inline README.md:52]: it should not be there?

File: settings.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 # Any of these can be overridden via environment variables with matching names
 class Settings(BaseSettings):
     # User or organization name, used for "user/image" -name generation
-    DOCKER_USER = "digitallivinginternational"  # Hopefully we'll get a shorter one
+    DOCKER_USER = "lietu"
```
Comment (fbjorn) [inline settings.py:7]: here as well

**Review by fbjorn — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (lietu) [reply README.md:6]: image

File: README.md (hunk 4 above)
Comment (lietu) [reply README.md:52]: image

File: settings.py (hunk 5 above)
Comment (lietu) [reply settings.py:7]: image

File: .github/workflows/build-and-upload.yaml (hunk 1 above)
Comment (lietu) [reply .github/workflows/build-and-upload.yaml:32]: good question

**Pull request merged**